client/src/components/month.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import moment from 'moment';
+import Day from './day.jsx';
+
+class Month extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            current: null
+        }
+    }
+
+    getDaysFormatted () {
```

Comment:
This function formats the days in the correct sequence based on the month I' passing in . The two most import function from moment are the "get # of days in month" and "get starting day of month"